.env.seed.large

```diff
@@ -1,3 +1,5 @@
+SEED_ADMIN_PASSWORD="Trying to fix hypertension."
```

Comment:
How about `trying to fix hypertension` to maintain consistency with review apps? Eg. Daniel may have built up some muscle memory.